### PR TITLE
added Administrate gem, created the dashboards, added star icons for the ratings in review

### PR DESCRIPTION
**Pull request by Alexboedtker**

### CODE REVIEW

**Comment by Alexboedtker** [comment]
Screenshot 2021-11-11 at 10 53 51

Screenshot 2021-11-11 at 11 00 05

**Comment by Alexboedtker** [comment]
Screenshot 2021-11-11 at 12 59 23

**Comment by Alexboedtker** [comment]
Here's a screen recording showing that the 5 star reviews get auto approved.

https://user-images.githubusercontent.com/71139736/141674314-bc59c99c-0f58-4f91-a2fe-dc3029b93f7c.mov

**Comment by Alexboedtker** [comment]
Here's a screenshot of the admin/reviews panel with the new methods approve and decline.
Screenshot 2021-11-14 at 09 49 28

**Comment by Alexboedtker** [comment]
Here's another screen recording to show that the approve and decline methods work.

https://user-images.githubusercontent.com/71139736/141674361-517bef08-f9ec-45fd-b839-a8b5b2036058.mov

File: config/routes.rb [hunk 1]
```diff
@@ -9,8 +9,8 @@
       root to: "users#index"
     end
 
-  patch 'admin/reviews/:id/approve', to: 'admin/reviews#approve', as: :approve_review
-  patch 'admin/reviews/:id/decline', to: 'admin/reviews#decline', as: :decline_review
+  patch 'admin/reviews/:id/approve', to: 'admin/reviews#approve', as: :admin_review_approve
```
Comment (inem) [inline config/routes.rb]: Let's add them inside "resources :reviews". Like [here](https://guides.rubyonrails.org/routing.html#adding-member-routes)

Comment (Alexboedtker) [reply config/routes.rb]: done and it still works :)

**Pull request closed without merging**